### PR TITLE
Update deployment target to iOS 13.

### PR DESCRIPTION
**Pull request by brandonpage**

### CODE REVIEW

**Review by bhariharan — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 anchors:
-  - &latest-xcode "11.7.0"
-  - &latest-ios   "13.7"
-  - &min-ios      "12.2"
-  - &beta-xcode   "12.0.0"
-  - &beta-ios     "14.0"
+  - &latest-xcode "12.1.0"
+  - &latest-ios   "14.1"
+  - &min-ios      "13.5"
```
Comment (bhariharan) [inline .circleci/config.yml:4]: `13.0`?

**Review by bhariharan — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (brandonpage) [reply .circleci/config.yml:4]: Blame Apple.  The Xcode 12.1 package only comes bundled with the 14.1 and 13.5 runtimes.

Comment (bhariharan) [reply .circleci/config.yml:4]: Gotcha.

**Review by bhariharan — APPROVED** [review]

**Comment by brandonpage** [comment]
Will send a new, clean PR.

**Pull request closed without merging**